libs/ledger-live-common/src/families/stellar/logic.ts

```diff
@@ -86,34 +183,36 @@ export const formatOperation = async (
     id: encodeOperationId(accountId, rawOperation.transaction_hash, type),
     accountId,
     fee: new BigNumber(transaction.fee_charged),
-    value,
+    value: rawOperation?.asset_code
+      ? new BigNumber(transaction.fee_charged)
+      : value,
```

Comment:
Does Ledger Live expect a fee as a value here for token accounts? 

This is causing an issue displaying operations on the Desktop and Mobile for accounts that no longer have USDC sub-account. Users can remove trustlines outside of Ledger Live.

![image](https://user-images.githubusercontent.com/7346473/173909698-2f6a3382-1b44-4145-8574-7a6a63d1afd3.png)